projects/ng-transcribe/src/lib/utils.ts

```diff
@@ -30,9 +30,9 @@ export class i18n {
         let result: string = ''
         let keys = key.split('.')
         if (keys.length > 1) {
-        result = this.getDepthValue(this.locale[this.conf.language], keys) || this.getDepthValue(this.locale[this.conf.fallback], keys)
+        result = this.conf.fallback ? this.getDepthValue(this.locale[this.conf.language], keys) || this.getDepthValue(this.locale[this.conf.fallback], keys) : this.getDepthValue(this.locale[this.conf.language], keys)
```

Comment:
this.locale[this.conf.fallback]
this.locale[this.conf.language]

repeated, can be moved out later on

Reply:
Addressed, @ukanirudh